src/CommandingOfficers/Ave.java

```diff
@@ -644,7 +644,7 @@ private static class OblidoAbility extends CommanderAbility
     private static final int OBLIDO_COST = 8;
     private static final int OBLIDO_BUFF = 20;
     private static final int OBLIDO_RANGE = 2;
-    private static final int OBLIDO_DAMAGE = 2;
+    private static final int OBLIDO_DAMAGE = 20;
```

Comment:
So is the rule "Always work in terms of health (0-100, typically) and only convert to HP when rendering"?

Reply:
Soooooort of.
The problem is that we have to handle (almost) everything as if the rounded value is the real one.
But yes, everything refers to raw percents now.